tests/splines/polynomial_evaluator.hpp

```diff
@@ -32,12 +32,13 @@ struct PolynomialEvaluator
             }
         }
 
-        double operator()(double const x) const noexcept
+        KOKKOS_FUNCTION double operator()(double const x) const noexcept
         {
             return eval(x, 0);
         }
 
-        void operator()(ddc::ChunkSpan<double, ddc::DiscreteDomain<DDim>> chunk) const
+        KOKKOS_FUNCTION void operator()(
+                ddc::ChunkSpan<double, ddc::DiscreteDomain<DDim>> chunk) const
```

Comment:
```suggestion
        void operator()(ddc::ChunkSpan<double, ddc::DiscreteDomain<DDim>> chunk) const
```
I would not make it a device function because the MemorySpace template parameter is fixed to `HostSpace` unless the function becomes template